priv/gettext/ar/LC_MESSAGES/default.po

```diff
@@ -412,7 +412,7 @@ msgid "Date"
 msgstr ""
 
 #, elixir-format
-#: lib/school_house_web/templates/layout/_lesson_menu.html.heex:312
+#: lib/school_house_web/templates/layout/_lesson_menu.html.heex:317
```

Comment:
I may be missing something, but why are these line numbers for "Debugging" translation changed in this pr? Does it fix an existing line mismatch?

Reply:
Yes, it fixes it:

https://github.com/lubien/school_house/blob/missing-translation/lib/school_house_web/templates/layout/_lesson_menu.html.heex#L317

Reply:
I've ran `mix gettext.extract` then `mix gettext.merge`

Reply:
🙌🏻 thanks!!